docs/topics/keyvault-secrets.md

```diff
@@ -0,0 +1,110 @@
+# Use Key Vault as the Source of Cluster Configuration Secrets

```

Comment:
I think some of this overlaps with https://github.com/Azure/aks-engine/blob/master/examples/keyvault-params/README.md

Reply:
It does. I've redirected the example api model readme doc to this new one.